public/src/components/forms/GroupForm.tsx

```diff
@@ -278,22 +278,22 @@ export class GroupForm extends React.Component<IGroupProps, IGroupState> {
         return flash;
     }
 
-    private isCurrentStudentSelected(student: IUserRelation): boolean {
+    private isCurrentStudentSelected(student: Enrollment): boolean {
         let foundSelected = false;
         this.state.selectedStudents.forEach((user) => {
-            if (user.user.getId() === student.user.getId()) {
+            if (user.getUserid() === student.getUserid()) {
                 foundSelected = true;
             }
         })
         return foundSelected;
     }
 
-    private getSelectedStudents(curUser: IUserRelation | undefined): IUserRelation[] {
-        const ss: IUserRelation[] = [];
+    private getSelectedStudents(curUser: Enrollment | undefined): Enrollment[] {
```

Comment:
The `curUser` argument is not used and can be removed.

Reply:
Fixed.